src/main/telemetry/telemetry.c

```diff
@@ -75,7 +75,9 @@ PG_RESET_TEMPLATE(telemetryConfig_t, telemetryConfig,
             IBUS_SENSOR_TYPE_TEMPERATURE,
             IBUS_SENSOR_TYPE_RPM_FLYSKY,
             IBUS_SENSOR_TYPE_EXTERNAL_VOLTAGE
-    }
+    },
+    .mavlink_mah_as_heading = false,
+    .mavlink_mah_as_heading_divisor = 10
```

Comment:
In general, when using comma separated lists, add a comma after the last entry - C will ignore it, and the end result is that in the next changeset that adds more entries, the last existing line will not show as changed, making reviewing easier.